specification/network/resource-manager/Microsoft.Network/stable/2020-03-01/privateEndpoint.json

```diff
@@ -567,6 +853,115 @@
+          "description": "A unique read-only string that changes whenever the resource is updated."
+        },
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource type."
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/PrivateDnsZoneGroupPropertiesFormat",
+          "description": "Properties of the private dns zone group."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "./network.json#/definitions/SubResource"
+        }
+      ],
+      "description": "Private dns zone group resource."
+    },
+    "PrivateDnsZoneGroupPropertiesFormat": {
+      "properties": {
+        "provisioningState": {
+          "readOnly": true,
+          "$ref": "./network.json#/definitions/ProvisioningState",
+          "description": "The provisioning state of the private dns zone group resource."
+        },
+        "privateDnsZoneConfigs": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PrivateDnsZoneConfig"
+          },
+          "description": "A collection of private dns zone configurations of the private dns zone group."
+        }
+      },
+      "description": "Properties of the private dns zone group."
+    },
+    "PrivateDnsZoneConfig": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Name of the resource that is unique within a resource group. This name can be used to access the resource."
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/PrivateDnsZonePropertiesFormat",
+          "description": "Properties of the private dns zone configuration."
+        }
+      },
+      "description": "PrivateDnsZoneConfig resource."
+    },
+    "PrivateDnsZonePropertiesFormat": {
+      "properties": {
+        "privateDnsZoneId": {
+          "type": "string",
+          "description": "The resource id of the private dns zone."
+        },
+        "recordSets": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/RecordSet"
+          },
+          "description": "A collection of information regarding a recordSet, holding information to identify private resources."
+        }
+      },
+      "description": "Properties of the private dns zone configuration resource."
+    },
+    "RecordSet": {
+      "properties": {
+        "recordType": {
+          "type": "string",
+          "description": "Resource record type."
+        },
+        "recordSetName": {
+          "type": "string",
+          "description": "Recordset name."
+        },
+        "fqdn": {
+          "type": "string",
+          "description": "Fqdn that resolves to private endpoint ip address."
+        },
+        "provisioningState": {
+          "readOnly": true,
+          "$ref": "./network.json#/definitions/ProvisioningState",
```

Comment:
where is ProvisioningState defined?

Reply:
..\azure-rest-api-specs\specification\network\resource-manager\Microsoft.Network\stable\2020-03-01\network.json